DOC: Simplify contributing intro (#54220)
pandas-dev/pandas: doc/source/development/contributing.rst

Getting started with Git
------------------------

`GitHub has instructions <https://docs.github.com/en/get-started/quickstart/set-up-git>`__ for installing git,
setting up your SSH key, and configuring git.  All these steps need to be completed before
you can work seamlessly between your local repository and GitHub.

.. _contributing.forking:

> Forking pandas

Reviewer: ```suggestion
Create a fork of pandas
```

> --------------

Reviewer: ```suggestion
-----------------------
```

> You will need your own fork to work on the code. Go to the `pandas project

Reviewer: ```suggestion
You will need your own copy of pandas (aka fork) to work on the code. Go to the `pandas project
```

> page <https://github.com/pandas-dev/pandas>`_ and hit the ``Fork`` button. You will

Reviewer: ```suggestion
page <https://github.com/pandas-dev/pandas>`_ and hit the ``Fork`` button. Please uncheck the box to copy only the main branch before selecting ``Create Fork``.
```

> want to clone your fork to your machine

Reviewer: ```suggestion
You will want to clone your fork to your machine
```
